src/ride/station.c

```diff
@@ -64,7 +64,10 @@ static void ride_update_station_blocksection(rct_ride *ride, int stationIndex)
 {
 	rct_map_element *mapElement;
 
+	assert(ride != NULL);
 	mapElement = ride_get_station_start_track_element(ride, stationIndex);
+	assert(mapElement != NULL);
+
```

Comment:
To fix the saved game that was posted on Reddit yesterday, I had to return if mapElement was NULL here - otherwise the park would just crash.